constants/src/lib.rs

```diff
@@ -239,6 +240,26 @@ pub const SQL_FN_NUM_POWER: u32 = 0x00100000;
 pub const SQL_FN_NUM_RADIANS: u32 = 0x00200000;
 pub const SQL_FN_NUM_ROUND: u32 = 0x00400000;
 
```

Comment:
The way I create these is actually pasting the defines from the header file and modifying them with regular expressions:
/'<,'>s/#define //
/'<,'>s/ /: u32 =/
/'<,'>s/^/pub const /
/'<,'>s/$/;/